packages/rpc-provider/src/substrate-connect/Health.ts

```diff
@@ -0,0 +1,332 @@
+// Copyright 2017-2022 @polkadot/rpc-provider authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+/* eslint-disable sort-keys */
+/* eslint-disable promise/param-names */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
>  It basically clobbers the rules for the repo which is there to ensure we don't run into unknowns.

We do have a good set of tests for ensuring that we don't run into those unknowns.

Also, what it tends to happen when enforcing these rules is that the code ends up being worse just for the purpose of shutting up the rules e.g (`.catch(console.log)`), etc. We use to have most of these rules on `substrate-connect` and we got rid of them because they were creating more issues than the issues that they were trying to solve.

> There are obviously places where this needs to apply despite best efforts (false positives, places where ordering is important, e.g. definitions, etc.), but generally we try to keep things consistent.

If you really care that much about having the keys sorted alphabetically, I will update the code accordingly... sure :shrug:

Reply:
All those global `eslint-disable` have now been removed :tada: